app/serializers/delivery_partners/participants_serializer.rb

```diff
@@ -14,10 +14,10 @@ def induction_record(participant_profile, delivery_partner)
       end
 
       def status_name(participant_profile, delivery_partner)
-        ParticipantProfileStatus.new(
+        StatusTags::DeliveryPartnerParticipantStatusTag.new(
```

Comment:
And here cc @DFE-Digital/lead-provider-cpd

Reply:
see other comment